src/WebHandlers.cpp

```diff
@@ -203,38 +207,45 @@ void AsyncStaticWebHandler::handleRequest(AsyncWebServerRequest* request) {
     // set etag to lastmod timestamp if available, otherwise to size
     String etag;
     if (lw) {
-      setLastModified(gmtime(&lw));
+      setLastModified(lw);
 #if defined(TARGET_RP2040)
       // time_t == long long int
-      const size_t len = 1 + 8 * sizeof(time_t);
+      constexpr size_t len = 1 + 8 * sizeof(time_t);
       char buf[len];
-      char* ret = lltoa(lw, buf, len, 10);
+      char* ret = lltoa(lw + request->_tempFile.size(), buf, len, 10);
```

Comment:
would it be better to do `lw ^ request->_tempFile.size()` ?

Reply:
yeah, agreed!